ansible/roles/dof_etherpad/templates/deployment.yml.j2

```
@@ -67,11 +67,24 @@ spec:
           name: apikey
         resources:
           limits:
-            cpu: {{ ETHERPA_CPU_LIMITS|default("1000m", true) }}
-            memory: {{ ETHERPA_MEMORY_LIMITS|default("1Gi", true) }}
+            cpu: {{ ETHERPAD_CPU_LIMITS|default("1000m", true) }}
+            memory: {{ ETHERPAD_MEMORY_LIMITS|default("1Gi", true) }}
           requests:
-            cpu: {{ ETHERPA_CPU_REQUESTS|default("100m", true) }}
-            memory: {{ ETHERPA_MEMORY_REQUESTS|default("128Mi", true) }}
+            cpu: {{ ETHERPAD_CPU_REQUESTS|default("100m", true) }}
+            memory: {{ ETHERPAD_MEMORY_REQUESTS|default("128Mi", true) }}
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: 9001
+          initialDelaySeconds: 60
+          periodSeconds: 10
+        livenessProbe:
+          httpGet:
+            path: /health
+            port: 9001
+          path: /health
```

fix for the http path form the livenessProb
```suggestion
        livenessProbe:
          httpGet:
            path: /health
            port: 9001
```